core-bundle/contao/templates/backend/be_two_factor.html5

```diff
@@ -152,7 +152,7 @@
                   <form method="post">
                     <input type="hidden" name="FORM_SUBMIT" value="tl_passkeys_credentials_actions">
                     <input type="hidden" name="REQUEST_TOKEN" value="<?= $this->requestToken ?>">
-                    <?php if ($this->editPassKeyId === $credential->getId()): ?><button type="submit" name="save_passkey" form="form-edit-passkey-<?= $credential->getId() ?>" value="<?= $credential->getId() ?>"><?= Contao\Image::getHtml('save.svg', $this->trans('MSC.edit')) ?></button><?php else: ?><button type="submit" name="edit_passkey" value="<?= $credential->getId() ?>"><?= Contao\Image::getHtml('edit.svg', $this->trans('MSC.edit')) ?></button><?php endif; ?> <button type="submit" name="delete_passkey" value="<?= $credential->getId() ?>" onclick="if(!confirm('<?= $this->trans('MSC.deleteConfirm', [$credential->name ?: $credential->getId()]) ?>'))return false"><?= Contao\Image::getHtml('delete.svg', $this->trans('MSC.delete')) ?></button>
+                    <?php if ($this->editPassKeyId === $credential->getId()): ?><button type="submit" name="save_passkey" form="form-edit-passkey-<?= $credential->getId() ?>" value="<?= $credential->getId() ?>" aria-label="<?= Contao\StringUtil::specialchars($this->trans('MSC.savePasskey', [$credential->name ?: $i + 1])) ?>"><?= Contao\Image::getHtml('save.svg', $this->trans('MSC.edit')) ?></button><?php else: ?><button type="submit" name="edit_passkey" value="<?= $credential->getId() ?>" aria-label="<?= Contao\StringUtil::specialchars($this->trans('MSC.editPasskey', [$credential->name ?: $i + 1])) ?>"><?= Contao\Image::getHtml('edit.svg', $this->trans('MSC.edit')) ?></button><?php endif; ?> <button type="submit" name="delete_passkey" value="<?= $credential->getId() ?>" onclick="if(!confirm('<?= $this->trans('MSC.deleteConfirm', [$credential->name ?: $credential->getId()]) ?>'))return false" aria-label="<?= Contao\StringUtil::specialchars($this->trans('MSC.deletePasskey', [$credential->name ?: $i + 1])) ?>"><?= Contao\Image::getHtml('delete.svg', $this->trans('MSC.delete')) ?></button>
```

Comment:
All the `aria-label` attributes on the button elements should be `alt` attributes on the icons IMHO. @ausi /cc

Reply:
No, that was already there.

This is the _back end_ template.

Reply:
Because while implementing the same feature for the front end, I noticed these shortcomings for the back end.